dashboard/app/controllers/activities_controller.rb

```diff
@@ -2,7 +2,7 @@
 require 'cdo/share_filtering'
 
 class ActivitiesController < ApplicationController
-  include LevelsHelper
+  include MilestoneHelper
```

Comment:
Weird to be including helpers directly in the controller... I think this may have the side effect of exposing all helper methods as controller actions.